kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -122,6 +128,14 @@
         return this.getChannelTitle(this.contentNode && this.contentNode.channel_id);
       },
     },
+    methods: {
+      titleMaxLines(content) {
+        if (!content.is_leaf) {
```

Comment:
This isn't blocking and the code works as expected, but a comment explaining that at least three lines should always be displayed on folder cards should be added above this line for clarifying purposes.